packages/@tinacms/datalayer/package.json

```diff
@@ -28,6 +28,26 @@
     "test": "jest --passWithNoTests"
   },
   "dependencies": {
+    "@octokit/auth-app": "^4.0.9",
+    "@octokit/graphql": "^5.0.5",
+    "@octokit/rest": "^19.0.7",
+    "encoding-down": "^7.1.0",
+    "fast-glob": "^3.2.5",
+    "fs-extra": "^9.0.1",
+    "glob-parent": "^6.0.2",
+    "graphql": "15.8.0",
+    "gray-matter": "^4.0.2",
+    "isomorphic-git": "^1.10.3",
+    "js-yaml": "^3.14.0",
+    "jsonpath-plus": "^6.0.1",
+    "level": "^7.0.1",
+    "leveldown": "^6.1.0",
```

Comment:
@jamespohalloran what's up with all these deps getting added?

Reply:
Good question, not sure how that happened, taking a look

Reply:
Some of those look like the pre-abstractlevel dependencies

Reply:
I think there may have been a funky merge. Just removed those